Automattic/google-docs-add-on
# Add Post Options

Add a toggle-able drawer to hold categories, tags, and a remove-site button

**Testing**

- Test data migration
    - Clear all site data
    - Deploy the `master` branch version
    - Add a site
    - Deploy this branch
    - Confirm that site displays correctly and adds categories
- Test multiple sites
    - Add tags on one site
    - Create draft
    - Add tags on second site
    - Create draft
    - Confirm no cross-polination
